home.html

```diff
@@ -206,8 +206,58 @@ <h2>"DJ Night @Felicity2020"</h2>
 								<h3>by Sai Vineesh C</h3>
 								<p>Month: March'20</p>
 							</div>
-						</div>
+						</div> -->
 					</div>
+					<script src="imageDetails.js"></script>
```

Comment:
Instead of keeping it as a script, better to store the `imageDetails` as a JSON file and fetch the object from the same.